src/OpenLoco/Input/ShortcutManager.cpp

```diff
@@ -207,10 +208,40 @@ namespace OpenLoco::Input::ShortcutManager
         Windows::Map::centerOnViewPoint();
     }
 
+    static loco_global<uint8_t, 0x00525FB0> _525FB0;
```

Comment:
This is a vehicle function. I suggest it is moved into the vehicle.cpp file then the widgetIndex's can be filled in as part of the vehicle pr.